src/rpc/mining.cpp

```diff
@@ -789,6 +808,92 @@ class submitblock_StateCatcher : public CValidationInterface
     }
 };
 
+static UniValue pprpcsb(const JSONRPCRequest& request) {
```

Comment:
I presume the command name is meaningful?

Reply:
progpowrpcsubmitblock